sdcm/nemesis.py

```diff
@@ -1803,6 +1803,26 @@ def disrupt_delete_10_full_partitions(self):
 
         self.run_deletions(queries=queries, ks_cf=ks_cf)
 
+    def disrupt_delete_overlapping_row_ranges(self):
+        """
+        Delete several overlapping row ranges in the table with large partitions.
+        """
+        self.verify_initial_inputs_for_delete_nemesis()
+        ks_cf = 'scylla_bench.test'
+        partitions_for_delete = self.choose_partitions_for_delete(10, ks_cf, with_clustering_key_data=True)
+        if not partitions_for_delete:
+            self.log.error('No partitions for delete found!')
+            return
```

Comment:
instead of `return`, please do `UnsupportedNemesis`

Reply:
@fgelcer are you sure? it might be only temporarily happen on test start.
and the same logic is already used here:
https://github.com/scylladb/scylla-cluster-tests/blob/0324942753d8e4a5b56963853a5cda8eb583707d/sdcm/nemesis.py#L1761
and here:
https://github.com/scylladb/scylla-cluster-tests/blob/0324942753d8e4a5b56963853a5cda8eb583707d/sdcm/nemesis.py#L1744

Reply:
it is kind of before taking any action... to decide on which partition we want to run these deletions, so it is in the start of the nemesis.
we can confirm what @enaydanov @fruch and @bentsi think about it too

Reply:
ping @bentsi @enaydanov

Reply:
ping @bentsi @enaydanov

Reply:
fixed anyway